Source/Dafny/Resolver.cs

```diff
@@ -16985,6 +16980,21 @@ class MethodCallInformation {
       bindings.AcceptArgumentExpressionsAsExactParameterList(actuals);
     }
 
+    private static string GetLocationInformation(List<Formal> formals, int index) {
```

Comment:
No need to change anything, but here I would prefer `IReadOnlyList` over `List`, since it's a less specific interface and lets us know the method is pure.

Reply:
When I initially wrote `IReadOnlyList`, it complained because it was missing an import.
So I thought about it a second time. Since we know that at both call sites the first arguments are `List<Formal>`(likewise, we know that the index is within bounds), and this method is private, it makes sense to reuse the same time. So I chose it this way.